x/auth/keeper/querier.go

```diff
@@ -10,24 +10,24 @@ import (
 )
 
 // NewQuerier creates a querier for auth REST endpoints
-func NewQuerier(k AccountKeeper) sdk.Querier {
+func NewQuerier(k AccountKeeper, cdc codec.JSONMarshaler) sdk.Querier {
```

Comment:
Should we be consistent with the parameter naming? 
```suggestion
func NewQuerier(k AccountKeeper, legacyQuerierCdc codec.JSONMarshaler) sdk.Querier {
```